ReactNative/groceryList/src/actions/ProductActions.js

```diff
@@ -23,4 +24,16 @@ export const productCreate = ({ name, count, brand }) => {
 			Actions.pop()
 		});
   };
+};
+
+export const productsFetch = () => {
+	const { currentUser } = firebase.auth();
+
+	return (dispatch) => {
+		firebase.database().ref(`/users/${currentUser.uid}/products`)
+			.on('value', snapshot => {
+				dispatch({ type: PRODUCTS_FETCH_SUCCESS, payload: snapshot.val() });
+
+		});
+	};
```

Comment:
I think we can improve spaces here to be consistent with the whole application

Reply:
I have changed the problem with the spaces/tabs in the PR 31. I will update this pr when the PR-31 be merged.